rocky/rocky/views/organization_member_list.py

```diff
@@ -31,29 +27,20 @@ class OrganizationMemberListView(
     template_name = "organizations/organization_member_list.html"
     permission_required = "tools.view_organization"
 
-    def get_queryset(self):
-        queryset = self.model.objects.filter(organization=self.organization)
-        if "client_status" in self.request.GET:
-            status_filter = self.request.GET.getlist("client_status", [])
-            queryset = queryset.filter(status__in=status_filter)
-
-        if "blocked_status" in self.request.GET:
-            blocked_filter = self.request.GET.getlist("blocked_status", [])
-            blocked_filter_bools = []
+    def get_member_filter_form(self):
+        return MemberFilterForm(self.request.GET)
```

Comment:
This method doesn't have many references and seems redundant. I think this statement should be inlined (instantiate in the dispatch handler and in the template context)